README.md

```diff
@@ -12,7 +12,7 @@ stakeholders can be confident about the direction the project is evolving in.
 
 ## How to submit an RFC
 
-1. Copy `template.md` into `text/YYYY-MM-DD-my-feature.md`.
+1. Copy `template.md` into `text/PRID-my-feature.md`.
```

Comment:
One doesn't know the PRID before submitting the PR. You can refer to the way stated in my old PR.
By the way, I suggest include the propose time in each RFC.

Reply:
I think create a pull request and correspondingly change filename is trivial and thus we don't have to mention it clearly which looks redundant.

One may include the propose time in the RFC but I don't think it is a strong requirement. The time can be traced by the original pull request and generally we don't require an exact time but about how long ago, e.g., recent? months ago? years ago?